server/models/post.js

```diff
@@ -0,0 +1,61 @@
+const db = require('../db_config/init')
+
+
+module.exports = class Thought {
+    constructor(data){
+        this.id = data.id;
+        this.pseudonym = data.pseudonym;
+        this.title = data.title;
+        this.thought = data.thought;
+    }
+
+
+    static create(title, pseudonym, thought) {
+        return new Promise (async (resolve, reject) => {
+            try {
+                let formData = db.query(`INSERT INTO post title, pseudonym, thought VALUES ($1 $2 $3) RETURNING *;`, [title, pseudonym, thought])
```

Comment:
The query you have written has some syntax errors. Here are the following errors: 
- When you insert, you specify the column names you are inserting into in order within brackets. The column names should reference what you have defined in your SQL setup file. Therefore, you need to add brackets before title and after thought. 
- Between each value, you need to add commas. 

```
let formData = db.query(`INSERT INTO post (title, pseudonym, thought) VALUES ($1, $2, $3) RETURNING *;`, [title, pseudonym, thought])
```